src/commands/workspace/inject.ts

```diff
@@ -48,11 +50,21 @@ export default class Inject extends Command {
     'listr-renderer': listrRenderer
   }
 
+  // Holds cluster CLI tool name: kubectl or oc
+  private command!: string
+
   async run() {
     const { flags } = this.parse(Inject)
     const notifier = require('node-notifier')
     const cheTasks = new CheTasks(flags)
 
+    // Initialize cluster CLI tool name
+    const command = getClusterClientCommand()
+    if (!command) {
+      throw new Error('No cluster CLI client is installed.')
```

Comment:
Is it better to throw the error inside `getClusterClientCommand` ?

Reply:
+1

Reply:
Done